### DISCUSSION TITLE
Enable tests directory for scan

### DISCUSSION THREAD

**Discussion by andrii-pez** [JetBrains/Qodana#107] [Q&A] [answered]
Hi there!
I use Qodana with `qodana.recommended` profile and noticed that `tests/` directory is ignored. How I can add it to scan list?

P.S. By the way, what is the correct way to extend the list of inspection tools when `qodana.recommended` profile is used? Just create my own profile or should I add needed inspections to `include` list of `qodana.yaml`?

**Comment by brichbash** [comment, marked as answer]
Hi,

`qodana.recommended` and `qodana.starter` profiles ignore tests. You can either create an inspection profile with "tests" included to analysis scope or use "Default" profile (you can set it via `--profile-name` flag or in `qodana.yaml`) and [exclude](https://www.jetbrains.com/help/qodana/qodana-yaml.html#Set+up+a+profile+by+the+path) all unnecessary inspections via `qodana.yaml`.
>what is the correct way to extend the list of inspection tools when qodana.recommended profile is used

You can extend the profile using `qodana.yaml` the way you mentioned or create a new profile of your own.

**Reply by tiulpin** [reply]
@andrii-pez to sum up: if you include the directories you want to be included, they always be included. 

So you **can use** `qodana.recommended` and **include** `tests/`

**Reply by andrii-pez** [reply]
> to sum up: if you include the directories you want to be included, they always be included.

I tried next option for `qodana.yaml`:
```yaml
version: "1.0"
linter: jetbrains/qodana-php:latest
profile:
  name: qodana.recommended
include:
  - name: All
    paths: 
      - tests
...
```

and it didn't work for me. So it seems `qodana.recommended` always exclude `tests/` and we can do nothing with it

**Reply by tiulpin** [reply]
AFAIR `All` works only for exclude 🤔
